sippy-ng/src/tests/Tests.js

```diff
@@ -57,20 +57,10 @@ export default function Tests(props) {
                     component={Link}
                     to={url}
                   />
-                  <Tab
-                    label="Tests by variant"
-                    value="details"
-                    component={Link}
-                    to={url + '/details'}
-                  />
```

Comment:
I'm not sure I want to remove this page. It may be a little buggy, but the view isn't shown anywhere else this way, e.g.:

e..g, https://psql-sippy.dptools.openshift.org/sippy-ng/tests/4.11/details?test=remains%20available

Reply:
This was the one that worried me the most, it's thousands and thousands of rows and seemed almost unusable. It did load however, so if you think it's useful I can restore.

Reply:
The above link loads almost instantly for me -- with a filter I think the page is pretty usable